content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -86,7 +169,28 @@ Create the authorization policy:
       rules:
       - from:
         - source:
-           ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
+            ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
+    EOF
+    {{< /text >}}
+
+1. Here is the same policy, but using `remoteIpBlocks` instead of `ipBlocks`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+            remoteIpBlocks: ["1.2.3.4", "5.6.7.0/24"]
     EOF
     {{< /text >}}
```

Comment:
Instead of making these remoteIpBlocks examples another step, you should put the two examples into a {{< tabset >}} in the previous step.
https://preliminary.istio.io/latest/about/contribute/shortcodes/#use-tabs

Reply:
Good point, I'll change it to a tab.